builder/dockerfile/builder.go

```diff
@@ -42,9 +43,12 @@ var validCommitCommands = map[string]bool{
 }
 
 const (
-	stepFormat = "Step %d/%d : %v"
+	stepFormat = "%s Step %d/%d : %v"
```

Comment:
Some people might grep the progress with ^Step, so I think %s should be appended to the tail of this string

Reply:
of couse,we can prepend any character or move it at the end of the line to limit any breaking change

Reply:
Yes, I agree. If we do this let move it after from the beginning  - so pehaps:
`
Step 1/3 (3 sec): RUN echo hi
`